src/Microsoft.DotNet.ProjectJsonMigration/ProjectDependencyFinder.cs

```diff
@@ -14,6 +14,17 @@ namespace Microsoft.DotNet.ProjectJsonMigration
 {
     public class ProjectDependencyFinder 
     {
+        public IEnumerable<ProjectDependency> ResolveProjectDependencies(IEnumerable<ProjectContext> projectContexts)
```

Comment:
please also add the `pre resolved-dependencies` param, this is used so you can avoid unresolved references that are actually csproj references